skema/program_analysis/CAST/matlab/tests/test_binary_operation.py

```diff
@@ -0,0 +1,50 @@
+from skema.program_analysis.CAST.matlab.matlab_to_cast import MatlabToCast
+from pathlib import Path
+
+# Test the CAST returned by processing the simplest MATLAB binary assignment
+
+# The source code for the single line assignment test is 'z = x + y;'
+#  SYNTAX TREE:
+#  node: assignment
+#    node: identifier 'z'
+#    node: =
+#    node: binary_operator
+#      node: identifier 'x'
+#      node: +
+#      node: identifier 'y'
+
+def test_binary_operation():
+    """ Tests whether the parser can handle a binary operation """
+    filepath = Path(__file__).parent / "data" / "binary_assignment.m"
+    print(f"Testing parse tree for {str(filepath)}")
+    parser = MatlabToCast(filepath)
+    cast = parser.out_cast
+
+    # there should only be one CAST object in the cast output list
+    assert len(cast) == 1  
+
+    head = cast[0].to_json_object()
+    nodes = head['nodes']
+
+    # there should be only one node in this single-line assignment test
+    assert len(nodes) == 1
+
+    body = nodes[0]['body']
+
+    # There should only be one module within the node body 
+    assert len(body) == 1
+
+    # Test the module dictionary for relevant key values
+    module = body[0]
+    assert module['node_type'] == 'Assignment'
```

Comment:
I think this is a fine way to access the CAST, but whenever you get a chance you should look through my test scripts to see how I access the CAST using the objects instead of the dictionary. 
Here's an example [of a test script for expressions](https://github.com/ml4ai/skema/blob/main/skema/program_analysis/tests/test_expression_cast.py)

Honestly the dictionary way might be cleaner, but I'm used to doing it with the objects.